submissions/lawalNurudeen/javascript/conditionals.js

```diff
@@ -0,0 +1,22 @@
+const num1 = 123;
+const num2 = 7;
+const diff = num1 - num2;
+
+const numbers = [];
+
+for (let i = 1; i <= diff; i++){
+    if(i % 3 === 0 && i % 5 === 0){
+        console.log("Software Developer");
+    } else if (i % 3 === 0) {
+        console.log("Software");
+        numbers.push("Software");
```

Comment:
There is no need pushing to an array

Reply:
I basically found the array to have easily and visibly captured all content of the loop..